.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         with:
           toolchain: ${{ matrix.toolchain }}
       - name: cargo doc (all features)
-        run: cargo doc --all --all-features --document-private-items
+        run: cargo doc --workspace --all-features --document-private-items
```

Comment:
Seems to me that `--workspace` is also the default in directories that contain a workspace -- so we could just leave it out and clean all these up? That would be my preference.

Reply:
Fixed to remove `--all` throughout, using the implicit `--workspace` default.